{
  "name": "uncaught-exception",
  "version": "6.1.0",
  "description": "Handle uncaught exceptions",
  "keywords": [],
  "author": "Raynos <raynos2@gmail.com>",
  "repository": "git://github.com/uber/uncaught-exception.git",
  "main": "uncaught",
  "homepage": "https://github.com/uber/uncaught-exception",
  "bugs": {
    "url": "https://github.com/uber/uncaught-exception/issues",
    "email": "raynos2@gmail.com"
  },
  "dependencies": {
    "error": "^7.0.1",
    "json-stringify-safe": "^5.0.1",
    "process": "^0.11.1",
    "xtend": "^4.0.0"
  },
  "collaborators": [
    {
      "name": "David ellis"
    },
    {
      "name": "Amos barreto"
    }
  ],
  "devDependencies": {
    "assert-tap": "^0.1.4",
    "fake-fs": "^0.5.0",
    "istanbul": "^0.3.15",
    "jshint": "^2.8.0",
    "opn": "^2.0.0",
    "tape": "^4.0.0",
    "time-mock": "^0.1.2",
    "uber-statsd-client": "1.3.2"
  },
  "scripts": {
    "test": "npm run jshint && npm run cover",
    "jshint": "jshint --verbose .",
    "cover": "istanbul cover --report cobertura --print none ./test/index.js",
    "view-cover": "istanbul report html && opn ./coverage/index.html"
  },
  "engine": {
    "node": ">= 0.8.x"
  }
}
